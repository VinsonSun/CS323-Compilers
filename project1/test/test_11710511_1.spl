int fun( {
    int a = 01;
}
