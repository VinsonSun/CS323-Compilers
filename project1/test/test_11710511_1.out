Error type B at Line 1: Missing closing parenthesis ')'
Error type A at Line 2: unknown lexeme 01
